webui/react/src/components/TaskActionDropdown.tsx

```diff
@@ -25,14 +25,22 @@ import css from './ActionDropdown.module.scss';
 import Link from './Link';
 
 interface Props {
+  children?: React.ReactNode;
```

Comment:
nit: This works, but you could instead do `PropsWithChildren<Props>` instead of including children in Props

Reply:
will do!